libs/deal-ticket/src/components/market-info/info-market.tsx

```diff
@@ -267,6 +309,15 @@ export const Info = ({ market }: InfoProps) => {
               market.tradableInstrument.instrument.product
                 .oracleSpecForTradingTermination.id,
           }}
+          link={
+            <Link
+              target="_blank"
+              href={`${VEGA_EXPLORER_URL}/oracles#${market.tradableInstrument.instrument.product.oracleSpecForTradingTermination.id}`}
+              className="text-ui dark:text-white text-black underline hover:underline hover:text-black-60 dark:hover:text-white-80"
```

Comment:
chore (blocking): Remove classnames please. As an aside. We should have to set text color everywhere on every element. Those styles should be inherited

Reply:
yeah, happy to remove them now. updated

Reply:
Sorry this should have read. "We should _not_ have to set text color everywhere..."